homeassistant/components/teslemetry/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/teslemetry",
   "iot_class": "cloud_polling",
   "loggers": ["tesla-fleet-api"],
-  "requirements": ["tesla-fleet-api==0.9.2", "teslemetry-stream==0.6.6"]
+  "requirements": ["tesla-fleet-api==0.9.2", "teslemetry-stream==0.6.10"]
```

Comment:
Please bump in separate PR

Reply:
I did, this probably just needs a rebase.
https://github.com/home-assistant/core/pull/137159